modules/ocf_irc/files/anope/global.conf

```diff
@@ -0,0 +1,24 @@
+# Most comments have been stripped for easier editing
+# The original comments can be found on anope's wiki:
+# https://wiki.anope.org/index.php/2.0/Configuration#Global
+
+service {
+	nick = "Global"
+	user = "services"
+	host = "services.irc.ocf.berkeley.edu"
```

Comment:
No, there's a few more changed, such as the global messages on power cycles, and things like that: https://wiki.anope.org/index.php/2.0/Configuration#Global